visualizations/frontend/filters/TTagsSelector.vue

```diff
@@ -285,10 +285,7 @@ export default {
         ? JSON.stringify(urlObject)
         : "";
       return this.setFilterValue("selected_items", urlValue);
-    },
-    removeFilter() {
-      this.unsetFilterValue("selected_items");
-    },
+    }, 750),
```

Comment:
Are we 100% sure about this magic number? Could we test with a smaller value to see if the result is the same?

Reply:
Man I need to not forget about PRs. I tested with 500 and couldn't get it to trigger but that could be me being uncoordinated. 🤷‍♀️  Happy to reduce it.